components/operator/helm/templates/licence-secret.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if (not (eq .Values.operator.licence.secret "")) }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: licence
```

Comment:
Can the name be formated with helpers as the deployment `  name: {{ include "operator.fullname" . }}`

Reply:
+1